src/common/thread_measurement.cpp

```diff
@@ -33,6 +40,29 @@ void StartMeasurementTask(void const *argument) {
             next_fs_cycle = now + 50;
         }
 
+        // sample filament autoload
```

Comment:
move all changed code from this file into filament_sensor.cpp
create static function `autoload_loop` and call it inside `fs_cycle();` faster cycle 50ms for autoload is irrelevant

Reply:
Solved